app/models.py

```diff
@@ -39,6 +39,42 @@ def save(self):
         db.session.add(self)
         db.session.commit()
 
+    def generate_token(self, user_id):
+        """ Generates the access token"""
+
+        try:
+            # set up a payload with an expiration time
+            payload = {
+                'exp': datetime.utcnow() + timedelta(minutes=1440),
```

Comment:
don't you think this a lot of time for a session to be active?

Reply:
it's validity is 24hours, although I changed it to 12 hours, to avoid the user logging in every few minutes